### PR TITLE
Use correctly processed options in build command

### PR DESCRIPTION
**Pull request by loosebazooka**
fixes https://github.com/GoogleContainerTools/jib/issues/3018

### CODE REVIEW

File: jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Build.java [hunk 1]
```diff
@@ -71,21 +72,17 @@
...
-
   /**
    * Returns a user configured Path to a buildfile and if none is configured returns jib.yaml in
-   * {@link #getContextRoot()}.
+   * {@link #contextRoot}.
    *
-   * @return a path to a bulidfile
+   * @return a path to a buildfile
    */
   public Path getBuildFile() {
```
Comment (chanseokoh) [inline jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Build.java:81]: Doesn't seem like it needs to be `public`. `@VisibleForTesting`?

**Review by chanseokoh — APPROVED** [review]
Interesting. It used to work, but I guess it broke at some point during refactoring.

File: jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Build.java (hunk 1 above)
Comment (loosebazooka) [reply jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Build.java:81]: oh right.

**Comment by loosebazooka** [comment]
> Interesting. It used to work, but I guess it broke at some point during refactoring.

Yeah probably because I never added any good integration tests :\

**Pull request merged**